integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -66,6 +66,17 @@ <h3 class="heading font-bold text-black">
                 </div>
             </div>
         </div>
+        <div class="mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+            <div class="p-4 rounded bg-water-500">
+                <h3 class="heading font-bold text-black">
+                    <i icon-name="settings" class="mr-2"></i> {% translate "URL" %}
+                </h3>
+            </div>
+            <div class="px-4 pb-4">
+                <label for="{{ form.website.id_for_label }}">{{ form.website.label }}</label>
+                {% render_field form.website %}
```

Comment:
I think we can move this field into the "General settings" box together with the "name" input field.